crates/router/tests/connectors/connector_auth.rs

```diff
@@ -39,18 +39,8 @@ pub struct ConnectorAuthentication {
     pub worldpay: Option<BodyKey>,
     pub worldline: Option<SignatureKey>,
     pub zen: Option<HeaderKey>,
-    pub hs_base_url: Option<String>,
-    pub hs_api_key: Option<String>,
-    pub hs_test_browser: Option<String>,
-    pub chrome_profile_path: Option<String>,
-    pub firefox_profile_path: Option<String>,
-    pub pypl_email: Option<String>,
-    pub pypl_pass: Option<String>,
-    pub gmail_email: Option<String>,
-    pub gmail_pass: Option<String>,
-    pub configs_url: Option<String>,
-    pub stripe_pub_key: Option<String>,
-    pub testcases_path: Option<String>,
+    #[serde(flatten)]
```

Comment:
Remove the `#[serde(flatten)]` and move the configuration variables under an `[automation_configs]` table in the TOML files. Using `#[serde(flatten)]` makes it more prone to errors when editing the configuration corresponding to selenium tests in TOML file.

Also, maybe name this `selenium_test` instead of `automation_configs`, since this is specific to UI tests.